service/src/main/resources/static/swagger/openapi-docs.yaml

```
@@ -17,187 +17,187 @@ servers:
   - url: ../
     description: Relative to the current swagger page
 tags:
-  - name: Entities
-    description: Entity APIs
+  - name: Records
+    description: Record APIs
   - name: Instances
   - description: Instance APIs
```

one small fix to eliminate validation errors:
```suggestion
  - name: Instances
    description: Instance APIs
```